packages/admin-api-client/src/types.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  OperationVariables,
+  Headers,
+  ClientResponse,
+  GraphQLClient,
+} from "@shopify/graphql-client";
+
+export interface AdminApiClientConfig {
```

Comment:
We can now use the common `ApiClient` types that's available in `graphql-client` to help us simplify our types 😄